common.go

```diff
@@ -2,27 +2,69 @@ package go_bagit
 
 import (
 	"fmt"
-	"runtime/debug"
+	"os"
+	"path/filepath"
 	"time"
 )
 
 var currentTime = time.Now()
 
+const version = "0.2.3-alpha"
+
 func GetSoftwareAgent() string {
 	const mod = "github.com/nyudlts/go-bagit"
 
-	info, ok := debug.ReadBuildInfo()
-	if !ok {
-		return fmt.Sprintf("go-bagit <https://%s>", mod)
-	}
+	//this is broken always returns "devel"
```

Comment:
If this is cruft, is there a reason not to delete it?

Reply:
it was an external contribution